Felienne/hedy
# [UI IMPROVEMENT] Improves UI of pre-join page to align with overall UI

**Description**
In this PR we clean-up the pre-join page when joining a class. Currently we have two different pages, a `class-prejoin.html` and a `class-already-joined.html`. They are almost identical and it makes no sense to keep track of two templates. We deleted the `class-alread-joined.html` page and differentiate through an additional boolean value `joined`. The texts are aligned in the middel and instead of a plain-text `<a>` we use buttons.

**Fix for**
This PR fixes #1766.

**How to test**
Navigate to a class and use the join link when both in the class and not in the class. Notice that the same template is rendered but the content is different.

**Screenshots**
![Schermafbeelding 2022-01-24 om 15 00 09](https://user-images.githubusercontent.com/38155424/150796506-3e164cab-484f-410c-a16a-81dc9bddc3e5.png)
![Schermafbeelding 2022-01-24 om 14 59 59](https://user-images.githubusercontent.com/38155424/150796514-273c3597-9fcf-4e28-80d1-64b5767e0172.png)
